dtfs-central-api/package.json

```diff
@@ -38,7 +38,7 @@
     "express-rate-limit": "^6.11.2",
     "moment": "^2.29.4",
     "mongo-dot-notation": "3.1.0",
-    "mongodb": "4.2.0",
+    "mongodb": "4.17.0",
```

Comment:
I'm a bit confused here: the description says we're downgrading to 4.2.0 but this looks like we're upgrading to 4.17.0